types/ui-grid/ui-grid-tests.ts

```diff
@@ -123,6 +123,8 @@ gridApi.core.handleWindowResize();
 gridApi.core.queueGridRefresh();
 gridApi.core.queueRefresh();
 gridApi.core.registerColumnsProcessor(colProcessor, 100);
+gridApi.core.getRowsByKey(true, "id", 42);
+gridApi.core.findRowByKey(true, 10, "42");
```

Comment:
It looks like CI is failing because of these calls to functions on `gridApi.core`.
The `getRowsByKey` and `findRowByKey` functions are defined under the Grid class, not `gridApi.core`.

Reply:
Thanks, that was it